python/defaults.cfg

```diff
@@ -0,0 +1,3 @@
+[main]
+# Directory to store the sqlite databases
+DATABASE_DIRECTORY: /tmp
```

Comment:
Let’s use something under /var instead of /tmp

Reply:
This is only a default for running the server locally in dev mode without gunicorn

Reply:
Thanks. Understood.